addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3523,25 +3523,24 @@ const SnippetOptionWidget = Widget.extend({
      * @param {SnippetOptionWidget|UserValueWidget} parentWidget
      * @returns {Promise}
      */
-    _renderXMLWidgets: function (parentEl, parentWidget) {
-        const proms = [...parentEl.children].map(el => {
+    _renderXMLWidgets: async function (parentEl, parentWidget) {
+        const proms = [...parentEl.children].map(async el => {
             const widgetName = el.tagName.toLowerCase();
-            if (!userValueWidgetsRegistry.hasOwnProperty(widgetName)) {
+            if (!Object.prototype.hasOwnProperty.call(userValueWidgetsRegistry, widgetName)) {
```

Comment:
Why that change?

Reply:
It was regarding that eslint warning https://eslint.org/docs/rules/no-prototype-builtins

Reply:
If you read the logic behind that rule, it seems overkill to consider it here, disable the warning :wink: 
In particular:
```
When Not To Use It

You may want to turn this rule off if your code only touches objects with hardcoded keys, and you will never use an object that shadows an Object.prototype method or which does not inherit from Object.prototype.
```